modules/openy_features/openy_node/modules/openy_node_landing/config/install/core.entity_view_display.node.landing_page.default.yml

```diff
@@ -19,6 +20,14 @@ targetEntityType: node
 bundle: landing_page
 mode: default
 content:
+  addthis:
```

Comment:
What about this one? Is it used on this display? Probably should be hidden.

Reply:
Same here